pkg/operator/certrotation/client_cert_rotation_controller.go

```diff
@@ -117,97 +116,24 @@ func (c CertRotationController) syncWorker() error {
 	return nil
 }
 
-func (c *CertRotationController) WaitForReady(stopCh <-chan struct{}) {
-	klog.Infof("Waiting for CertRotationController - %q", c.name)
-	defer klog.Infof("Finished waiting for CertRotationController - %q", c.name)
-
-	if !cache.WaitForCacheSync(stopCh, c.cachesToSync...) {
-		utilruntime.HandleError(fmt.Errorf("caches did not sync"))
-		return
+func (c CertRotationController) targetCertRecheckerPostRunHook(ctx context.Context, syncCtx factory.SyncContext) error {
```

Comment:
pointer receiver

Reply:
@sttts do we need one? (i'm fine adding it)